lib/c_glib/test/CMakeLists.txt

```diff
@@ -108,13 +108,6 @@ add_test(NAME testoptionalrequired COMMAND testoptionalrequired)
 
 include_directories("${PROJECT_SOURCE_DIR}/test/c_glib/src" "${CMAKE_CURRENT_BINARY_DIR}/gen-c_glib")
 
-add_executable(testthrifttest testthrifttest.c
```

Comment:
Thrift test should be there. Iif I removed it it was because error. I will readd it.